kore/src/Kore/Repl.hs

```diff
@@ -54,6 +54,9 @@ import qualified Kore.AST.Common as Kore
                  ( Variable )
 import           Kore.AST.MetaOrObject
                  ( MetaOrObject )
+import           Kore.Attribute.Axiom
+                 ( LineColumn (..), Location (..), Source (..), location,
```

Comment:
Locatiuon and source are fields that do not use the type name as prefix, so, according to our style guide, they should be in a qualified import.